starlette/schemas.py

```diff
@@ -18,45 +24,90 @@ def render(self, content: typing.Any) -> bytes:
         assert isinstance(
             content, dict
         ), "The schema passed to OpenAPIResponse should be a dictionary."
+
+        def represent_dict(dumper, instance):
+            return dumper.represent_mapping("tag:yaml.org,2002:map", instance.items())
+
+        yaml.add_representer(collections.OrderedDict, represent_dict)
+        if LazyDict is not None:
```

Comment:
Actually I'd prefer it if `spec.to_dict()` just returned regular data structures, rather than making us jump through hoops.

I'd suggest:

```python
def make_plain_datastructure(data):
    if isinstance(data, list):
        return [make_plain_datastructure(item) for item in data]
    elif isinstance(data, dict):
        return {key: make_plain_datastructure(value) for key, value in data.items()}
    return data
```

Then have `get_schema` return `return make_plain_datastructure(self.spec.to_dict())`

Reply:
Yeah, this looks much better

Thanks!